azkaban-web-server/src/main/java/azkaban/webapp/servlet/LoginAbstractAzkabanServlet.java

```diff
@@ -677,4 +680,30 @@ protected void handleMultiformPost(final HttpServletRequest req,
       final HttpServletResponse resp, final Map<String, Object> multipart, final Session session)
       throws ServletException, IOException {
   }
+
+  /**
+   * Method to check permission to access image management APIs.
+   *
+   * @param imageTypeName
+   * @param user
+   * @param type
+   * @return boolean
+   */
+  protected boolean hasPermission(final String imageTypeName, final User user,
```

Comment:
hasPermission in this servlet seems very generic name. We need to mention what it handles. It is only for image types. so hasPermissionForImageTypeMgmt something?